source/services/index.md

```diff
@@ -0,0 +1,105 @@
+An `Ember.Service` is a long-lived `Ember.Object` available application-wide. Services are singletons, and only one instance of a given service can exist at a time.
+
+Ember itself uses services to provide access to the router, and Ember Data does the same with the shared store.
+
+Other example uses of services include:
+
+* Logging
+* User/session authentication
+* Geolocation
+* Third-party APIs
+* Web Sockets
+* Server-sent events or notifications
+* Server-backed API calls that may not fit Ember Data
+
+Because `Ember.Service` doesn't add any sugar to `Ember.Object`, services are not required to follow as many conventions as components or routes. Anything that can be accomplished with an `Ember.Object` can be stored in a service.
+
+### Defining Services
+
+To define a service, extend the `Ember.Service` base class:
+
+```app/services/shopping-cart.js
+export default Ember.Service.extend({
+});
+```
+
+Like any `Ember.Object`, a service can have properties and methods of its own.
```

Comment:
Definitely agree it is worth having `carts.` used somewhere in the JS. Even if it's a built-in CPM. @elbeezy and I discussed this at length and couldn't come up with an example that was simple and idiomatic (things like `isEmpty` belong in the service, not the component). Suggestions welcome.